moto/awslambda/policy.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import unicode_literals
+
+import json
+import uuid
+
+from six import string_types
+
+from moto.awslambda.exceptions import PreconditionFailedException
+
+
+class Policy:
+    def __init__(self, parent):
+        self.revision = str(uuid.uuid4())
+        self.statements = []
+        self.parent = parent
+
+    def __repr__(self):
+        return json.dumps(self.get_policy())
+
+    def wire_format(self):
+        return json.dumps(
+            {
+                "Policy": json.dumps(
+                    {
```

Comment:
Can we re-use the get-policy method here?
It looks like the JSON is exactly the same

Reply:
wire_format returns JSON and the `Policy` must be escaped JSON so it's somewhat different, however, I agree I was looking at that the other day. We'll take a look and see if we can refactor it a bit.